style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-color: rgb(141, 107, 56);
```

Comment:
As a way to set some defaults for your whole HTML document, I recommend that every `body` ruleset include the following:
- `background` or `background-color` ✅ you've done this!
- `color`
- `font-family`

You can always change these defaults on individual elements or specific classes elsewhere in your stylesheet, but adding them here creates a solid foundation and will make sure your document displays as expected across browsers.